demosplan/DemosPlanCoreBundle/Logic/Statement/TagService.php

```diff
@@ -77,20 +77,16 @@ public function getTag($id)
     /**
      * Creates a new Tag with the given title.
      *
-     * @param string $title
-     *
      * @throws DuplicatedTagTitleException
-     * @throws Exception
      */
-    public function createTag($title, TagTopic $topic, bool $persistAndFlush = true): Tag
+    public function createTag(string $title, TagTopic $topic, bool $persistAndFlush = true): Tag
     {
         $procedureId = $topic->getProcedure()->getId();
         if ('' === $title) {
             throw new InvalidArgumentException('Tag title may not be empty.');
         }
 
-        $titleCount = $this->tagRepository->count(['id' => $procedureId, 'title' => $title]);
-        if (0 !== $titleCount) {
+        if ($this->tagRepository->isTagTitleFree($procedureId, $title)) {
```

Comment:
```suggestion
        if (!$this->tagRepository->isTagTitleFree($procedureId, $title)) {
```

Reply:
only tiny thing here, if the tag exists, isTagTitleFree will return false and the exception will not be thrown otherwise seems good, i've tested too and it works good with the tiny fix